pontoon/base/urls.py

```diff
@@ -30,6 +30,15 @@
     url(r'^contributor/(?P<email>[\w.%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,4})/$',
         RedirectView.as_view(url="/contributors/%(email)s/", permanent=True)),
 
+    # List all active localization teams
+    url(r'^teams/$',
```

Comment:
I wonder if maybe `/locale/sl/` is a better URL structure, as the _thing_ we're dealing with is really a locale rather than a locale team.